inst/extdata/example_release.yaml

```diff
@@ -0,0 +1,57 @@
+
+spatial: "example_shapefile/"
+
+attr-table: "example_attributes.csv"
+
+parent-content: "example_parent.yaml"
+
+authors: ["Alison P Appling","Jordan S Read","Luke A Winslow", "Arroita", "Emily Bernhardt", "Natalie Griffiths", "Robert Hall", "Judson Harvey", "Heffernan", "Emily Stanley", "Edward Stets", "Charles Yackulic"]
+pubdate: 2017
+title: >-
+  Data release: A national database for river metabolism (2007-present)
+doi: http://dx.doi.org/10.5066/REPLACE.w.REAL
+abstract: >-
+  Abstract text from paper
+
+purpose: To bring metabolism data to the world!
+start-date: SCRIPT 20070101
+end-date: SCRIPT 2016XXXX
+update: update planned for year 2020
+themekeywords: ["water","streams","rivers","metabolism","primary productivity","carbon","respiration"]
+usage-rules: >-
+  These data are subject to change and are not citable until reviewed and approved for official publication by the USGS
+
+contact-person: Jordan S Read
```

Comment:
Also, this "person" is defined twice, on a more complex project it could be more.  It might be worth adding some sort of reference system to manage these details.